arch/x86/core/efi.c

```diff
@@ -33,8 +33,8 @@ void efi_init(struct efi_boot_arg *efi_arg)
 		return;
```

Comment:
Nit: typo on commit message on "respectively". Maybe only worth fixing if there are more stuff to change.